service/src/main/scala/za/co/absa/logingw/rest/service/InfoService.scala

```diff
@@ -18,10 +18,11 @@ package za.co.absa.logingw.rest.service
 
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Service
+import za.co.absa.logingw.rest.config.BaseConfig
 
 @Service
-class InfoService @Autowired()() {
+class InfoService @Autowired()(baseConfig: BaseConfig) {
   def getInfoMessage: String = {
-    s"Basic info message to be here." // todo service version, some parts of configuration perhaps - Issue #15
+    s"Basic info message to be here. '${baseConfig.someKey}'" // todo service version, some parts of configuration perhaps - Issue #15
```

Comment:
The info service should indicate what ${baseConfig.someKey} means to the user.

The Service is currently just a shell, so not a big deal at the moment.

Reply:
As you mentioned , since the service is currently just a shell, this is just a showcase. Can be updated on next feature PRs.